app/commands/payments/paypal/debug.rb

```diff
@@ -6,8 +6,11 @@ class Payments::Paypal::Debug
   def call
     return unless Rails.env.production?
 
-    IO.write("/mnt/efs/repos/debug/paypal.txt", "#{message}\n", mode: 'a')
+    IO.write("/mnt/efs/repos/debug/paypal.txt", "#{timestamp} #{message}\n", mode: 'a')
```

Comment:
```suggestion
    IO.write("/mnt/efs/repos/debug/paypal.txt", "#{timestamp} #{message}\n\n", mode: 'a')
```

(Add some breathing room for us in reading?)